docs/src/main/sphinx/security/user-mapping.rst

```diff
@@ -2,67 +2,123 @@
 User mapping
 ============
 
-User mapping defines rules for mapping from users in the authentication system to Trino users.  This
-mapping is particularly important for Kerberos or certificate authentication where the user names
-are complex like ``alice@example`` or ``CN=Alice Smith, OU=Finance, O=Acme, C=US``.
+User mapping defines rules for mapping from users in the authentication method to Trino users. This
+mapping is particularly important for :doc:`Kerberos </security/kerberos>` or
+certificate authentication where the user names
+are complex, such as ``alice@example`` or ``CN=Alice Smith, OU=Finance, O=Acme, C=US``.
 
-User mapping can be configured with a simple regex extraction pattern, or more complex rules in a
-separate configuration file.
+There are two ways to map the username format of a given authentication
+provider into the simple username format of Trino users:
+
+* With a single regular expression (regex) :ref:`pattern mapping rule <pattern-rule>`
+* With a :ref:`file of regex mapping rules <pattern-file>` in JSON format
```

Comment:
Should have periods at the end of each line item.

Reply:
Style guide says no, because no verbs. (https://developers.google.com/style/periods#bulleted-lists)

Reply:
Another rule implied by the thin statement in the Google Style Guide is that if one bullet has two sentences, and therefore has closing periods, then all members of the list have periods. But it's better to structure lists as far as possible with parallel structure so that list members are as alike as possible. Then they would either all have closing periods or all not.

Reply:
Ahh, good points.